Makefile

```diff
@@ -27,50 +28,33 @@ setup: check-commands
 # ------------------------------------------------------------
 #  Development
 # ------------------------------------------------------------
-# Starts all services
-start:	COMMANDS=docker
-start:	check-commands
-	docker compose -f docker/docker-compose.yml up
 
-stop:
-	docker compose -f docker/docker-compose.yml down
+start: check-commands
+	docker compose --profile $(DOCKER_PROFILE) -f docker/docker-compose.yml up -d
 
-start.nats:	COMMANDS=docker
-start.nats: check-commands
-	docker run -p 4222:4222 -p 8222:8222 -p 6222:6222 \
-	--mount type=bind,source="$$(pwd)"/crates/fuel-core-nats/nats.conf,target=/etc/nats/nats.conf \
-	--name fuel-core-nats-server \
-	-ti nats:latest --js --config /etc/nats/nats.conf
+stop: check-commands
+	docker compose --profile $(DOCKER_PROFILE) -f docker/docker-compose.yml down
 
-stop.nats:
-	docker rm -f $$(docker ps -a -q --filter ancestor=nats:latest)
+restart: stop start
 
-# Starts fuel-core-nats service
-start.fuel-core-nats:
-	./scripts/start-fuel-core-nats.sh
+clean/docker: stop
+	docker compose --profile $(DOCKER_PROFILE) -f docker/docker-compose.yml down -v --rmi all --remove-orphans
 
-dev-watch:
-	cargo watch -- cargo run
-
-# ------------------------------------------------------------
-# Build & Release
-# ------------------------------------------------------------
+start/nats stop/nats restart/nats clean/nats: DOCKER_PROFILE = nats
+start/fuel-core stop/fuel-core restart/fuel-core clean/fuel-core: DOCKER_PROFILE = fuel
 
-build: install
-	cargo build --release --target "$(TARGET)" --package "$(PACKAGE)"
+start/nats start/fuel-core: start
+stop/nats stop/fuel-core: stop
+restart/nats restart/fuel-core: restart
+clean/nats clean/fuel-core: clean/docker
```

Comment:
Nit: Maybe we don't need the extra `clean` aliases?

Reply:
This clean is nice to cleanup Docker sometimes, mine storage is almost full 😂